cli/get-multidomain-certificate.md

```diff
@@ -0,0 +1,64 @@
+---
+currentMenu: cli-get-multidomain-certificate
+---
+
+# Multi-domains certificates
+
+Let's Encrypt and most ACME servers are able to provide multi-domain certificates. Such certificates
+will be usable for multiple domains as a single file, which can be useful in many cases (for instance to
+use the same certificate for `yourdomain.org` and `www.yourdomain.org`).
+
+**Note:** Multi-domains certificates use a technology called [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)
+that is not available in every browser. Nowadays, almost all browsers support it, but in some contexts it may be a problem.
+Don't hesitate to check for yourself [the support of SNI](https://en.wikipedia.org/wiki/Server_Name_Indication#Support).
+
+## Obtain a multi-domain certificate
+
+The process of obtaining a multi-domain certificate is almost the same as
+[getting one for a single domain](documentation/cli/get-certificate.html).
+
+The only difference is when you use the `request` command, to add the other domains you want to support:
+
+``` console
+php acmephp.phar request yourdomain.org -a www.yourdomain.org -a anotherdomain.org
+```
+
+**Note**: you need to be the authorized owner of all the domains you request this way. In this case, it means
+you have to run `php acmephp.phar authorize yourdomain.org`, `php acmephp.phar authorize www.yourdomain.org`
+and `php acmephp.phar authorize anotherdomain.org` before being able to request the multi-domains certificate.
+
+The process will be the same as with a simple certificate: the command will ask you some informations, request
+the certificate to the ACME server and store it under the `~/.acmephp` directory.
+
+The first domain provided will be the main one for Acme PHP, meaning the multi-domain certificate will be stored
```

Comment:
Technicaly not the first, given there is 1 Domain and N alternativeDomains.

By the ways, that's means, we can't manage 2 certificates for the same domain with differents alternative names. The last generated will override the previous